app/controllers/benefits/CompanyVanFuelBenefitsController.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
```

Comment:
Can you update the app dependencies and rerun tests

 uk.gov.hmrc:play-frontend-hmrc to 1.19.0-play-28
 uk.gov.hmrc.mongo to 0.55.0